locales/ara.json

```diff
@@ -313,8 +321,7 @@
       },
       "bleError": {
         "title": "فشل التحويل",
-        "message": "حدث خطأ أثناء نقل بطاقة . حاول مرة اخرى.",
-        "hint": "خطأ: {{code}}"
+        "message": "حدث خطأ أثناء نقل {{vcLabel}}. حاول مرة اخرى."
```

Comment:
We are not using {{vclabel}} placeholder now, please replace it with the "card" value in arabic.
ref : https://github.com/tw-mosip/inji/pull/56

Reply:
same case for all languages.

Reply:
Thanks, I'll make the suggested change